src/cpp/rtps/transport/shared_mem/SharedMemManager.hpp

```diff
@@ -855,6 +878,26 @@ class SharedMemManager :
             return ret;
         }
 
+        bool recover_blocked_processing()
```

Comment:
The only place where this is used is ignoring the returned value. Consider returning void